app/views/docs/show.html.erb

```diff
@@ -12,6 +12,8 @@
   <%= @doc.description %>
 </p>
 
+author: <%= @doc.user.name if @doc.user %>
```

Comment:
eu ainda vou melhorar essa parte. To esperando @bernardogfilho me passar umas manhas do framework de front.